packages/traceability-schemas/package.json

```diff
@@ -10,11 +10,12 @@
     "postinstall": "npm run build",
     "lint": "eslint . --fix",
     "test": "node --max-old-space-size=4096 'node_modules/.bin/jest'",
-    "regenerate": "npm i && node ./scripts/regenerate.js",
+    "regenerate": "node ./scripts/regenerate.js",
+    "count-issuer-dependent": "node ./scripts/count-issuer-dependent.js",
     "build:openapi": "node ./scripts/schemas-to-openapi.js",
     "build:context": "node ./scripts/openapi-to-context.js",
     "build:vocab": "node ./scripts/schemas-to-vocab.js",
-    "build": "npm run build:openapi && npm run build:context && npm run build:vocab"
+    "build": "npm run build:openapi && npm run build:context && npm run build:vocab && npm run regenerate && npm run count-issuer-dependent"
```

Comment:
We don't need the JSOn-LD tests, because they duplicate the logic that runs on install.